openshift/distro/centos/7/Dockerfile

```diff
@@ -111,12 +111,11 @@ RUN source scl_source enable ${RUBY_SCL} \
       yum -y install ${BUILD_DEP_PKGS} ; \
     fi \
  && bundle install --deployment --jobs $(grep -c processor /proc/cpuinfo) \
-# Bundler < 1.12.0 needs fixing for git gems with extensions (Puma)
- && if ruby -e "begin; require 'rubygems'; Gem::Specification.find_by_name('bundler').version < Gem::Version.new('1.12.0') && exit(0); rescue; end; exit(1)"; then \
-      echo "[WARNING] Old Bundler requires fixing git gems with extensions (Puma)" ; \
-      PUMA_DIR=$(bundle show puma) \
-      && ln -s $(find "${PUMA_DIR}" -name puma_http11.so) "${PUMA_DIR}"/lib/puma ; \
-    fi \
+
+ ## bundler does not place the .so in the correct dir
```

Comment:
just one `#` is enough ;)

Reply:
this is rather weird - something from the ruby configuration, rubygems or bundler (or even puma using some wrong directory?) might be changing from a local installation (non-RHSCL) which I can't reproduce

Reply:
I can't reproduce it either on a non-rhscl env.